### ISSUE TITLE
bug for apollo reference hardware page

### ISSUE THREAD

**Issue by Wsine** [ApolloAuto/apollo#1749]
the hardware page shows here http://apollo.auto/platform/hardware.html

the can card image is 
![image](http://apollo.auto/images/platform/hardware/CAN_e9fc326.png)

the page wrote model `C.2045.02` while the real model is `C.2045.04`

**Comment by YangHan101** [comment]
Already made the correction and updated the website. Thank you very much for pointing it out!

**Comment by Wsine** [comment]
I saw the update. But it make me confused that the page update to this one `CAN-PCIe/402-B4 part # C.2045.08` while the [installation_guide.md](https://github.com/ApolloAuto/apollo/blob/master/docs/quickstart/apollo_1_5_hardware_system_installation_guide.md#controller-area-network-can-card) refers to `CAN-PCIe/402`. Which one is the real?

the hardware page:
![](http://apollo.auto/images/platform/hardware/CAN_15ddb4b.png)

the installation guide:
![](https://github.com/ApolloAuto/apollo/raw/master/docs/quickstart/images/can_card.png)

**Comment by Capri2014** [comment]
@YangHan101

**Comment by YangHan101** [comment]
@Wsine They both should work. What matters is how many CAN channels you need. The CAN card in the top image is a 4-channel version. The one at the bottom is a single channel version.

Back in Apollo 1.0, we only needed one CAN channel to communicate with the vehicle CAN bus. So we used the one at the bottom. It was cheaper, too.

Later on, we added the radar which is also a CAN device. So we used the multichannel version from ESD. 

In short, you can pick the right number of CAN channels for you.

**Comment by Zhenni17** [comment]
Closing the issue, and feel free to reopen if I missed anything.
